modules/pot.py

```diff
@@ -0,0 +1,39 @@
+"""The Pot class, which is the quantity of chips that have been better.
+
+The pot is a dictionary of chip: quantity pairs.
+The pot will have all chip denominations populated, even if quantity is zero.
+"""
+
+from chip import Chip
+
+
+class Pot:
+    def __init__(self):
+        self.chips = ({
+             '50': 0,
+             '25': 0,
+             '10': 0,
+             '5': 0,
+             '1': 0
+        })
```

Comment:
An idea that might make dealing with this class much easier: that what specific denominations of chips are in your pot is actually just a visual detail. Store just a single int in here, then in your `__str__()` convert to actual chip counts (it would be just like the Change Return practice problem!). None of the actual gameplay logic cares about what kind of chips you have, right? Just the total value of your bet?